modules/gdpr_tasks/src/Anonymizer.php

```diff
@@ -124,73 +97,24 @@ public function run(TaskInterface $task) {
     // Make sure we load a fresh copy of the entity (bypassing the cache)
     // so we don't end up affecting any other references to the entity.
     $user = $task->getOwner();
-
     $errors = [];
-    $entities = [];
-    $successes = [];
-    $failures = [];
-    $log = [];
 
     if (!$this->checkExportDirectoryExists()) {
       $errors[] = 'An export directory has not been set. Please set this under Configuration -> GDPR -> Right to be Forgotten';
```

Comment:
Can we make this actually link to the right page?

Reply:
Done.